tests/imagegpt/test_modeling_imagegpt.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         hidden_act="gelu",
         hidden_dropout_prob=0.1,
         attention_probs_dropout_prob=0.1,
-        max_position_embeddings=512,
+        max_position_embeddings=1024,
```

Comment:
`ImageGPTFeatureExtractor` will create sequences of length 32 * 32 = 1024.
Using `max_position_embeddings=512` will give index out of range when dealing with position embeddings.

Reply:
Great to know about this 💯  Thank you!